docs/init.md

```diff
@@ -0,0 +1,93 @@
+
+    Binary:              /usr/local/bin/stacks-node
+    Configuration file:  /etc/stacks-blockchain/Config.toml
+    Data directory:      /stacks-blockchain
+    PID file:            /run/stacks-blockchain/stacks.pid
+    Lock file:           /var/lock/subsys/stacks (SysVinit)
+
+The PID directory and data directory should both be owned by the
+stacks user and group. It is advised for security reasons to make the
+configuration file and data directory only readable by the stacks user and
+group.
+
+NOTE: When using the systemd .service file, the creation of the aforementioned
+directories and the setting of their permissions is automatically handled by
+systemd. Directories are given a permission of 710, giving the stacks group
+access to files under it _if_ the files themselves give permission to the
+stacks group to do so. This does not allow for the listing of files under the directory.
+
+```bash
+$ mkdir -p /etc/stacks-blockchain/
+$ mkdir -p /stacks-blockchain
+$ useradd stacks
+$ chown -R stacks:stacks /stacks-blockchain/
+$ chgrp -R stacks /etc/stacks-blockchain
+```
+
+### macOS
+
+    Binary:              /usr/local/bin/stacks-node
+    Configuration file:  /etc/stacks-blockchain/Config.toml
+
+The Config.toml file is presumed to have group ownership by the `wheel` group, with the current user having membership to that group. The data directory defined in the config is required to be writable by the current user. Logs (stdout & stderr) are saved under `/tmp/stacks-blockchain.log`
+
+## Installing Service Configuration
+
+### systemd
+
+Installing this .service file consists of just copying it to
+/usr/lib/systemd/system directory, followed by the command
+`systemctl daemon-reload` in order to update running systemd configuration.
+
+To test, run `systemctl start stacks` and to enable for system startup run
+`systemctl enable stacks`
+
+NOTE: When installing for systemd in Debian/Ubuntu the .service file needs to be copied to the /lib/systemd/system directory instead.
+
+### SysVinit
+
+Copy stacks.init to /etc/init.d/stacks. Test by running `service stacks start`.
+
+Using this script, you can adjust the config path and log location to the stacks-node program by
+setting the STACKS_BLOCKCHAIN_CONFIG and STACKS_BLOCKCHAIN_LOG environment variables in the file
+/etc/sysconfig/stacks-blockchain.
+
+### MacOS
+
+Copy org.stacks.stacks-blockchain.plist into ~/Library/LaunchAgents. Load the launch agent by
+running `launchctl load ~/Library/LaunchAgents/org.stacks.stacks-blockchain.plist`.
+
+This Launch Agent will cause the stacks-blockchain to start whenever the user logs in.
+
+NOTE: This approach is intended for those wanting to run stacks-blockchain as the current user.
+You will need to modify org.stacks.stacks-blockchain.plist if you intend to use it as a
+Launch Daemon with a dedicated stacks user.
+
+## Auto-respawn
+
+Auto respawning is currently only configured for systemd.
```

Comment:
It's also probably not something you'd want in the first place, since if the node does not safely shut down, its chainstate could be left in an inconsistent state.

Reply:
I will set the service to not restart for now - but we could also set it to restart *only* if the shutdown returns 0. 
I'm not aware of what an unclean shutdown returns today, but we may be able to enable this in the future.

Reply:
Updated the doc to specifically state that auto respawning is disabled (as well as updated the startup scripts to not auto-restart)